events/test_events_db.py

```diff
@@ -3,12 +3,12 @@
 import mongomock
 import app
 
-
 EXAMPLE_TIME = datetime.datetime(2019, 6, 11, 10, 33, 1, 100000)
 
 
 class TestEventsDB(unittest.TestCase):
     def setUp(self):
+        self.maxDiff = None
```

Comment:
I'm not sure I see where this is currently used.

Reply:
Oops! This was for testing and I forgot to remove it, thanks for catching it